cirq/google/xmon_gates.py

```diff
@@ -14,9 +14,8 @@
 
 """Gates that can be directly described to the API, without decomposition."""
 
-from typing import Dict, Optional, Union, Tuple, Any, cast
+from typing import Dict, Union, Tuple, Any, cast
```

Comment:
nit: alphabetical

Reply:
Acknowledged. Leaving as is because this file is not long for this world.